.prettierignore

```diff
@@ -0,0 +1,3 @@
+build
+node_modules
+.github
```

Comment:
remove .github

Reply:
It can remain so that prettier ignores it